### DISCUSSION TITLE
what's the most trivial way to bundle external files for macos?

### DISCUSSION THREAD

**Discussion by Sani-m** [tauri-apps/tauri#4902] [Q&A]
What is the "most" trivial way to bundle i.e. *sh files for mac os builds, I still want to release a *single* executable...

Maybe I'm missing something obvious in the tauri.conf.json?

**Comment by abisar** [comment]
If they're just shell scripts, you can use [process::Command](https://doc.rust-lang.org/std/process/struct.Command.html) struct to execute commands and all it needs is a string, so no external .sh files. Here is an example:
```rust
use std::process::Command;

// Your awesome script, and you can put your scripts in another .rs file to organize
let script = format!{r#"
    Year=`date +%Y`;
    Month=`date +%m`;
    echo `date`;
    echo "Current year is: $Year";
    echo "Current month is: $Month";
    "#};

let script_output = Command::new("sh")
    .args(["-c", &script])
    .output()
    .expect("could not run shell");
```
You have lots of control over it such as stdin, stdout, flags, signals, etc.
I'm just not sure if `sh -c` accepts multiline.
